src/app/app.component.ts

```diff
@@ -127,5 +174,34 @@ export class AppComponent {
         console.log(e);
         this.setStatus('Error sending coin; see log.');
       });
+
+    this.pong.deployed.sendCoin(receiver, amount, {from: this.pong.getYourAccount()});
+    this._ethObservable.refresh(this._pongService);
   };
 }
+export enum ContractEnum {
+  META_COIN
+}
+export class PongStaticData {
+  constructor(public pongval: string) {
+  }
+}
+export interface PongDeployed extends TruffleContract {
+  sendCoin(receiver, amount, from),
+  getBalance
+}
+
+
+export class PongContract implements ContractValues<PongDeployed, PongStaticData> {
```

Comment:
Why is it called Pong? lol does that mean something?

Reply:
No, I've copied from https://github.com/PawelPisarek/truffle-angular-starter/blob/master/src/app/contract-sender/pong.service.ts. These class is only for join static-data and deployed data. It's my mistake forgetting rename.